deploy/crds/csi.ibm.com_configs_crd.yaml

```diff
@@ -2,6 +2,10 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: configs.csi.ibm.com
+  labels:
+    app.kubernetes.io/name: ibm-block-csi-operator
+    app.kubernetes.io/instance: ibm-block-csi-operator
```

Comment:
I think the values of these two labels should be according to the name of the specific yaml.
e.g. in this case, `app.kubernetes.io/name` should be `configs` or something

Reply:
My change was good enough to solve the ICP lint issue.
no idea what kind of error I will get for other values.

Reply:
well, I don't think these values are correct.
would it be possible to check?

Reply:
done